flash/video/classification/data.py

```diff
@@ -111,6 +127,84 @@ def predict_load_sample(self, sample: Dict[str, Any]) -> Dict[str, Any]:
         return self._encoded_video_to_dict(EncodedVideo.from_path(sample[DefaultDataKeys.INPUT]))
 
 
+class VideoClassificationFiftyOneDataSource(FiftyOneDataSource):
```

Comment:
I was starting to implement this, but then it seemed weird that it would then inherit from the `PathDataSource` and from the `FiftyOneDataSource` which seemed like it could cause issues since they are different `DataSource` subclasses